lib/geocoder/lookups/olleh.rb

```diff
@@ -113,6 +113,8 @@ def self.check_query_type(query)
         'convert_coord'
       when options.include?(:l_code)
         'addr_step_search'
+      when options.include?(:sr) && options.include?(:places)
+        'addr_local_search'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.